plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py

```diff
@@ -67,12 +93,21 @@ def uiCreate(cls, parentLayout, relativeToWhat):
         kFileOptionsStr = getMayaUsdString("kFileOptions")
         kMakePathRelativeStr = getMayaUsdString("kMakePathRelativeTo" + relativeToWhat)
         kMakePathRelativeAnnStr = getMayaUsdString("kMakePathRelativeTo" + relativeToWhat + "Ann")
+        kUnresolvedPathStr = getMayaUsdString("kUnresolvedPath")
+        kUnresolvedPathAnnStr = getMayaUsdString("kUnresolvedPathAnn")
+        kResolvedPathStr = getMayaUsdString("kResolvedPath")
+        kResolvedPathAnnStr = getMayaUsdString("kResolvedPathAnn")
  
         optBoxMarginWidth = mel.eval('global int $gOptionBoxTemplateDescriptionMarginWidth; $gOptionBoxTemplateDescriptionMarginWidth += 0')
         cmds.setParent(topForm)
         cmds.frameLayout(label=kFileOptionsStr, collapsable=False)
         widgetColumn = cmds.columnLayout()
-        cmds.checkBox(cls.kMakePathRelativeCheckBox + relativeToWhat, label=kMakePathRelativeStr, ann=kMakePathRelativeAnnStr)
+        cmds.checkBox(cls.kMakePathRelativeCheckBox, label=kMakePathRelativeStr, ann=kMakePathRelativeAnnStr)
```

Comment:
Since there can only be a single dialog no need to have differently named check boxes.